erroranalysis/erroranalysis/_internal/version_checker.py

```diff
@@ -0,0 +1,15 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+import pandas as pd
+from warnings import warn
+
+
+def check_pandas_version(feature_names):
```

Comment:
maybe add some method doc, even though it's under internal?

Reply:
Done